packages/social-auth/src/GoogleAuth.ts

```diff
@@ -1,11 +1,15 @@
 /* istanbul ignore file */
 import { Auth } from "./Auth";
-import type { IDP } from "./types";
+import type { IDP, RedirectSurface } from "./types";
 
 export class GoogleAuth extends Auth {
   idpName: IDP = "google";
   clientId = "1070572364808-d31nlkneam5ee6dr0tu28fjjbsdkfta5.apps.googleusercontent.com";
 
+  constructor(redirectSurface: RedirectSurface) {
```

Comment:
this is inherited, so please remove it
<img width="562" alt="Screenshot 2024-06-27 at 11 19 36" src="https://github.com/trilitech/umami-v2/assets/129749432/3fd354cd-fdb6-41f8-a7b8-06ee3c47cab2">

Reply:
Thank you! Fixed